charts/shoot-core/components/charts/vpn-shoot/templates/vpn-networkpolicy.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: {{ include "networkpolicyversion" . }}
+kind: NetworkPolicy
+metadata:
+  name: gardener.cloud--allow-vpn
+  namespace: {{ .Release.Namespace }}
+spec:
+  egress:
+  - {}
```

Comment:
Does it make sense to restrict egress to the shoot networks (pod/service/node CIDRs)?

Reply:
Yeah that would make sense. Does the `vpn-shoot` need connection to anything outside of these networks?

Reply:
AFAIK only those networks, nothing else. cc @mvladev @DockToFuture